setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="confluent-kafka-helpers",
-    version="0.10.5",
+    version="1.0.0",
```

Comment:
I think this library deserves a stable release version, what do you say? :smile: